custom_components/tuya_local/devices/zym201_presence_sensor.yaml

```diff
@@ -58,7 +58,7 @@ secondary_entities:
           max: 550
         mapping:
           - scale: 100
-            step: 100
+#            step: 100

```

Comment:
Where did the 100 come from in the first place?  Does the device accept being set to other values than 1.5, 2.5, 3.5, 4.5, 5.5?  If not maybe a select is better for this setting.